types/stripe-js/elements/address.d.ts

```diff
@@ -0,0 +1,243 @@
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the element gains focus.
+   */
+  on(
+    eventType: 'focus',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'focus',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'focus',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the element loses focus.
+   */
+  on(
+    eventType: 'blur',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'blur',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'blur',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the escape key is pressed within the element.
+   */
+  on(
+    eventType: 'escape',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'escape',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'escape',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the element fails to load.
+   */
+  on(
+    eventType: 'loaderror',
+    handler: (event: {elementType: 'address'; error: StripeError}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'loaderror',
+    handler: (event: {elementType: 'address'; error: StripeError}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'loaderror',
+    handler?: (event: {elementType: 'address'; error: StripeError}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Triggered when the loader UI is mounted to the DOM and ready to be displayed.
+   */
+  on(
+    eventType: 'loaderstart',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  once(
+    eventType: 'loaderstart',
+    handler: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+  off(
+    eventType: 'loaderstart',
+    handler?: (event: {elementType: 'address'}) => any
+  ): StripeAddressElement;
+
+  /**
+   * Updates the options the `AddressElement` was initialized with.
+   * Updates are merged into the existing configuration.
+   */
+  update(options: Partial<StripeAddressElementOptions>): StripeAddressElement;
+};
+
+export interface ContactOption {
+  name: string;
+  phone?: string;
+  address: {
+    line1: string;
+    line2?: string;
+    city: string;
+    state: string;
+    postal_code: string;
+    country: string;
+  };
+}
+
+export type AddressMode = 'shipping' | 'billing';
+
+export interface StripeAddressElementOptions {
+  /**
+   * Control which mode the AddressElement will be used for.
+   */
+  mode: AddressMode;
+
+  /**
+   * Control which countries are displayed in the address Element.
```

Comment:
Nit: `address Element` to `AddressElement`

Question: Should we include something like `Two-letter ISO code representing the country` in this comment and other areas that relate to country?

Reply:
Updated to use `Payment Element` and added `Two-letter ISO code` for this field.